api/src/test/java/org/pmiops/workbench/api/ConceptsControllerTest.java

```diff
@@ -141,6 +153,14 @@ public void testSearchConceptsBlankQuery() throws Exception {
             new SearchConceptsRequest().query(" ")));
   }
 
+  @Test
+  public void testSearchConceptsMultipleWordQuery(){
```

Comment:
Can you add another test with multiple words that actually returns something, and asserts those results are correct?